go/vt/vtgate/engine/replace_variables.go

```diff
@@ -80,7 +80,7 @@ func (r *ReplaceVariables) Inputs() []Primitive {
 }
 
 // Description implements the Primitive interface
```

Comment:
Some places we still have this comment, in others we do not. I do not have strong opinions about this. I am okay with it being the way it is or having it the same everywhere.